scss/_button-group.scss

```diff
@@ -126,7 +126,7 @@
 
   > .btn:not(:first-child),
   > .btn-group:not(:first-child) {
-    margin-top: calc(#{$btn-border-width} * -1); // stylelint-disable-line function-disallowed-list
```

Comment:
You'll have to [interpolate the Sass variables](https://sass-lang.com/documentation/interpolation/) across all of these `calc()` functions.

Reply:
oof... it looks like that is a requirement for node-sass in the failed CI, which uses LibSass (both deprecated in favor of Dart Sass).

See <https://sass-lang.com/documentation/syntax/special-functions/#element-progid-and-expression>
<img width="746" alt="" src="https://github.com/twbs/bootstrap/assets/20387688/327d7ce4-5556-4bf9-b7b3-70d4b77498ba">
> *LibSass*, Ruby Sass, and versions of Dart Sass prior to 1.40.0 *parse calc() as special syntactic function* like element().
> Dart Sass versions 1.40.0 and later parse calc() as a [calculation](https://sass-lang.com/documentation/values/calculations).

Does Bootstrap support LibSass builds in v5? I know that it was internally dropped but I mean for when people import and build the Sass themselves.

Reply:
Yeah, v5 still supports it. We'll drop it in v6 I imagine.

Reply:
This would probably have to hold off until then, because the calc simplification only happens when the variables are not interpolated. Switching everything to calc would probably make the output too large in most cases.

Reply:
I agree in terms of realistic timeline that we could consider dropping LibSass in v5.4/v5.5. I hope, however, that it wouldn't trigger a semver comments war.